tools/hatch-cli/src/templates/client-sdk/command.ts

```diff
@@ -0,0 +1,16 @@
+import commander from 'commander';
+import {clientSDKCreator, projectCreator, runCommander} from '../../util';
+
+commander
+  .name('hatch client-sdk')
+  .option('-n, --name <name>', 'The client SDK package name. Defaults to <dependency>-sdk if ' +
+    'generating from a dependency, else required', null)
+  .option('-d, --dependency <dependency>', 'The hatch microservice or webapp dependency name to ' +
+    'generate the OpenAPI spec', null)
+  .option('-v, --ver <ver>', 'The hatch microservice or webapp dependency version to generate the ' +
+    'OpenAPI spec. Defaults to \"latest\"', null)
+  .option('-i, --input <input>', 'The location of the OpenAPI spec, as URL or file', null)
```

Comment:
per offline discussion, let's use a more descriptive name here instead of `input`

Reply:
updated